web-client/public/js/graph.js

```diff
@@ -1049,11 +1049,12 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
                     var largeArc = 0;  // 1 or 0
                     var sweep = 1;     // 1 or 0
                     var offset = parseFloat(d.strokeWidth);
+                    var nodeShift = 1.033;
```

Comment:
Can the initial value of `nodeShift`, `1.033`, be given a name? Even something like `INITIAL_NODE_SHIFT` or `DEFAULT_NODE_SHIFT` sounds appropriate. We can change to using `const` when/if we move to ES6.

Reply:
No problem, I changed it. I tried to match the other variables under the comment // Defaults for normal edge. when I first named it.